input/fsh/VRCL/harmonized-profiles/ObservationMethodOfDisposition_new.fsh

```diff
@@ -0,0 +1,18 @@
+Profile: ObservationDecedentDispositionMethodVitalRecordsNew
+Parent: Observation
+Id: Observation-decedent-disposition-method-vr-new
+Title: "Decedent Disposition Method"
+Description: "Decedent Disposition Method (Observation)"
+//* status 1..1 loosening this restriction as BFDR does not have status 
+//* status only code
+//* status = #final 
+* code 1..1
+* code only CodeableConcept
+* code = $loinc#80905-3 // "Body disposition method"
+* subject 1..1
+* subject only Reference(Decedent) //BFDR references Patient - Decedent Fetus - Vital Records, which is a USCorePatient. Need to consolidate?
```

Comment:
BFDR references a Fetus, VRDR references a Decedent.  Both are derived from USCorePatient.
So, this profile should reference USCorePatient.